Documentation/installation/k0s.rst

```diff
@@ -0,0 +1,99 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _k0s_install:
+
+*****************************
+Installation k0s Using k0sctl
+*****************************
+
+This guide walks you through installation of Cilium on `k0s <https://k0sproject.io/>`_,
+an open source, all-inclusive Kubernetes distribution, 
+which is configured with all of the features needed to build a Kubernetes cluster.
+
+Cilium is presently supported on amd64 and arm64 architectures.
+
+Install a Master Node
+=====================
+
+The first step (after installing the k0sctl binary obviously) is to prepare the yaml configuration file k0sctl will use:
```

Comment:
Please explicitly request the user to install k0sctl, something like:

```suggestion
Ensure you have the k0sctl binary installed locally.

Prepare the yaml configuration file k0sctl will use:
```

The first sentence should give a link to the install docs upstream. It could be placed above the "Install a Master Node" title.